content/docs/rfcs/10/README.md

```diff
@@ -295,70 +316,82 @@ The following are **not** considered as part of the adversarial model:
+Personally Identifiable Information (PII) refers to any piece of data that could potentially be used to uniquely identify a party.
+For example, the signature verification key, and 
+the hash of one's static IP address are unique for each user and hence count as PII.
+Notice that users' actions can be traced through their PIIs 
+(e.g., signatures) and hence result in their re-identification risk.
+As such, we seek anonymity by avoiding linkability between actions and 
+the actors / actors' PII. 
+Concerning anonymity, Waku v2 provides the following features:
 
 **Publisher-Message Unlinkability**:
-This feature signifies the unlinkability of a publisher to its published messages in the 11/WAKU2-RELAY protocol.
-The [Publisher-Message Unlinkability](/spec/11#security-analysis) is enforced through the `StrictNoSign` policy due to which the data fields of pubsub messages that count as PII for the publisher must be left unspecified.
+This feature signifies the unlinkability of a publisher to its published messages in the [11/WAKU2-RELAY](/spec/11) protocol.
+The [Publisher-Message Unlinkability](/spec/11#security-analysis) is enforced through the `StrictNoSign` policy,
+due to the data fields of pubsub messages that count as PII for the publisher must be left unspecified.
 
 **Subscriber-Topic Unlinkability**:
-This feature stands for the unlinkability of the subscriber to its subscribed topics in the 11/WAKU2-RELAY protocol.
+This feature stands for the unlinkability of the subscriber to its subscribed topics in the [11/WAKU2-RELAY](/spec/11) protocol.
 The [Subscriber-Topic Unlinkability](/spec/11/#security-analysis) is achieved through the utilization of a single PubSub topic.
-As such, subscribers are not re-identifiable from their subscribed topic IDs as the entire network is linked to the same topic ID.
-This level of unlinkability / anonymity is known as [k-anonymity](https://www.privitar.com/blog/k-anonymity-an-introduction/) where k is proportional to the system size (number of subscribers).
-Note that there is no hard limit on the number of the pubsub topics, however, the use of one topic is recommended for the sake of anonymity. 
-
-### Spam protection
-
-This property indicates that no adversary can flood the system (i.e., publishing a large number of messages in a short amount of time), either accidentally or deliberately, with any kind of message i.e. even if the message content is valid or useful.
-Spam protection is partly provided in `11/WAKU2-RELAY` through the [scoring mechanism](https://github.com/libp2p/specs/blob/master/pubsub/gossipsub/gossipsub-v1.1.md#spam-protection-measures) provided for by GossipSub v1.1.
-At a high level, peers utilize a scoring function to locally score the behavior of their connections and remove peers with a low score.
+As such, subscribers are not re-identifiable from their subscribed topic IDs (`pubsub_topic`) as the entire network is linked to the same topic ID (`pubsub_topic`).
```

Comment:
I am assuming topic IDs refers to `pubsub_topic` .

Reply:
Correct.